roles/search/molecule/default/converge.yml

```diff
@@ -1,10 +1,15 @@
 ---
 - name: Converge
   hosts: all
-  tasks:
-    - name: "Include search"
-      include_role:
-        name: "search"
-      vars:
-        search_environment: # This will add extra vars at the end of the solr.in.sh file
-          SOLR_JAVA_MEM: "-Xms700m -Xmx900m"
+  vars:
+    search_environment: # This will add extra vars at the end of the solr.in.sh file
+      SOLR_JAVA_MEM: "-Xms700m -Xmx900m"
+  pre_tasks:
+    - name: generate a shared secret
+      set_fact:
+        reposearch_shared_secret: >-
+          {{ lookup('ansible.builtin.password', '/dev/null chars=ascii_letters,digits') }}
```

Comment:
we probably want something that would not change between reruns?

Reply:
I don't see how that could be a problem for molecule tests

Reply:
for local development I could run molecule converge multiple times, and also on the CI we could have some re-run on failure policy. This would mean having a secret change in every run that also lead to a service restart, I would try to avoid that.

I also remember we had something that automatically set all secrets in molecule, don't remember what/where it was unfortunately

Reply:
not sure why that is a problem but I changed to a static string